plugins/misp/plugin.spec.yaml

```diff
@@ -157,18 +170,22 @@ types:
     event:
       title: Event ID
       description: Event ID to append to
+      example: "1"
       type: string
     text:
       title: Text
       description: Text
+      example: "Example text"
       type: string
     comment:
       title: Comment
       description: Attribute comment
+      example: "Example attribute comment"
```

Comment:
Can you please add a realistic comment for attribute - may be "Updating {attribute}"